airflow/providers/elasticsearch/hooks/elasticsearch.py

```diff
@@ -23,12 +23,15 @@
 
 from deprecated import deprecated
 from elasticsearch import Elasticsearch
+from elasticsearch.client import SqlClient
```

Comment:
This import causes deprecation warning
https://github.com/elastic/elasticsearch-py/blob/4ee46509e236bf30f540d173b48bb7650159556d/elasticsearch/client.py#L75-L80

Reply:
Indeed...We should access the SQL client directly via es instance. I'll file a PR to fix it later